### PR TITLE
Add 'Julian Vu' to list of alumni

### PR DESCRIPTION
**Pull request by julianvu**

### CODE REVIEW

File: alumni.html [hunk 1]
```diff
@@ -46,6 +46,9 @@
   <!--End of Menu section-->
 
 <h3>STEM Alumni</h3>
+<ol>
+  <li>Julian Vu</li>
```
Comment (julianvu) [inline alumni.html:50]: Why did you add this line?

**Review by khoanguyendinh — APPROVED** [review]
